packages/core/components/FileList/useFileAccessContextMenu.ts

```diff
@@ -19,6 +21,46 @@ export default function useFileAccessContextMenu(filters?: FileFilter[], onDismi
     const userSelectedApplications = useSelector(interaction.selectors.getUserSelectedApplications);
     const { executionEnvService } = useSelector(interaction.selectors.getPlatformDependentServices);
 
+    const [hasPlateBarCode, setHasPlateBarCode] = React.useState(false);
+    let plateUILink: string | undefined = undefined;
+
+    fileSelection.fetchAllDetails().then((fileDetails) => {
+        console.log(fileDetails);
+        const annotations = fileDetails[0].annotations;
+        const platebarcode = annotations.find((x) => x.name === "Plate Barcode");
```

Comment:
This should be a constant defined [here (you'd need to add it to it first)](https://github.com/AllenInstitute/aics-fms-file-explorer-app/blob/master/packages/core/constants/index.ts#L16)

Reply:
not understanding, Is this refering to the plateUI links?? or something else

Reply:
I believe SEan means the "Plate Barcode" part should be a constant. So the change would look like `x.name === AnnotationName.PLATE_BARCODE` after you add `PLATE_BARCODE` to the constants file